src/main/java/com/autotune/analyzer/recommendations/RecommendationConstants.java

```diff
@@ -318,6 +323,8 @@ public static final class NotificationCodes {
         public static final int INFO_LONG_TERM_RECOMMENDATIONS_AVAILABLE = 111103; // TODO: need to discuss the code;
         public static final int INFO_COST_RECOMMENDATIONS_AVAILABLE = 112101;
         public static final int INFO_PERFORMANCE_RECOMMENDATIONS_AVAILABLE = 112102;
+        public static final int INFO_MODEL_RECOMMENDATIONS_AVAILABLE = 112103;
```

Comment:
This needs to be updated in the [MD file](https://github.com/kruize/autotune/blob/master/design/NotificationCodes.md) as well